aptos-move/framework/aptos-names/sources/config.move

```diff
@@ -199,6 +215,16 @@ module aptos_names::config {
         set_v1(@aptos_names, config_key_domain_price(length), &price)
     }
 
+    public entry fun set_public_key(sign: &signer, public_key: vector<u8>) acquires ConfigurationV1 {
```

Comment:
Is this an administration public key? If so, should this be more clearly renamed to `set_admin_public_key`?

Reply:
yup will do